src/scripting/lua_map.cc

```diff
@@ -3295,8 +3347,8 @@ int LuaTrainingSiteDescription::get_food_attack(lua_State* L) {
 /* RST
    .. attribute:: food_defense
 
-      (RO) A table of tables with food ware names used for Defense training,
-           e.g. {{"barbarians_bread"}, {"fish", "meat"}}
+      (RO) A :class:`table` of tables with food ware names used for Defense training,
```

Comment:
The training attributes sometimes start with Uppercase and sometimes with lowercase here and below. This should be made consistent.

Reply:
made all lowercase